<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModInstallingForm</class>
 <widget class="QWidget" name="ModInstallingForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="imageLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Resources/main.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="modNameLabel">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>HeadacheApache</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="descriptionLabel">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Мод добавляет в игру 3 персонажа которые смогут прочесть по 1 конкретной книги которые вы им принесете, персонажей можно найти в городе Солитьюд в вашем жилище под название: «Высокий шпиль» мод является демонстрацией.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="flagsVerticalLayout">
     <item>
      <widget class="QLabel" name="modulNameLabel">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Modul Name</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="nextPushButton">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
